.github/workflows/publish-helm-chart.yml

```diff
@@ -78,7 +78,7 @@ jobs:
           - name: Build Helm Chart
             id: build-chart
             run: |
-              chart_tarball=$(helm package ${{matrix.chartpath}})
+              chart_tarball=$(helm package ${{matrix.chartpath}} | awk '{print $NF}')
```

Comment:
```suggestion
              # Extract tarball name from a line that looks like
              # Successfully packaged chart and saved it to: /home/blin/vcs/images/osg-helm-charts/supported/osg-htc/osdf-origin/osdf-origin-0.29.tgz
              helm_package_success_message=$(helm package ${{matrix.chartpath}} | grep -m1 '^Success')
              chart_tarball=${helm_package_success_message#*': '}  # delete up to and including ': '
              if [[ ! $chart_tarball ]]; then
                  echo "helm package failed" >&2
                  exit 1
              fi
```
The 'awk' command is vulnerable to whitespace in the tarball path (unlikely but still), and also the return code of the statement is the return code of 'awk' so it swallows up errors.

Reply:
Much better!